test-automation/api-tests/test.py

```diff
@@ -67,34 +69,45 @@ def load_keys() -> None:
         if not os.path.isdir(f"{args.keys_dir}/{key}"):
             continue
         key_content = os.listdir(f"{args.keys_dir}/{key}")
-        new_account_name = f"_Test_key_{random_num}_{i}"
-        CLI.remove_account(new_account_name)
-        for f in key_content:
-            if f.startswith("."):
+        account_name = f"{ACC_NAME_PREFIX}{random_num}_{i}"
+        CLI.remove_account(account_name)
+        for file_name in key_content:
+            if not file_name.endswith(".key"):  # Strong assumption about key file, some valid files may be ignored.
                 continue
-            key_file_path = f"{os.path.abspath(args.keys_dir)}/{key}/{f}"
+            from_key_file_path = f"{os.path.abspath(args.keys_dir)}/{key}/{file_name}"
+            to_key_file_path = f"{CLI.keystore_path}/{account_name}"
+            if not os.path.isdir(to_key_file_path):
+                os.mkdir(to_key_file_path)
+            shutil.copy(from_key_file_path, to_key_file_path)
             try:
-                response = CLI.single_call(f"keys import-ks {key_file_path} {new_account_name} "
-                                           f"--passphrase={args.passphrase}").strip()
-                if f"Imported keystore given account alias of `{new_account_name}`" == response:
-                    ACC_NAMES_ADDED.append(new_account_name)
-                    break
-            except RuntimeError:
-                pass  # It's okay if import fails, just try next file in key's dir.
+                address = CLI.get_address(account_name)
+                names_with_address = CLI.get_accounts(address)
+                if len(names_with_address) > 1:  # Remove duplicate accounts as passphrase may fail.
```

Comment:
This logic is there b/c we can have duplicate keys in the keystore (possibly with 2 different pw)